test-utils/state-viewer/src/state_dump.rs

```diff
@@ -42,16 +42,77 @@ pub fn state_dump(
         })
         .collect::<HashMap<_, _>>();
 
-    let mut total_supply = 0;
+    let mut near_config = near_config.clone();
 
-    let mut records_path_dir = records_path.to_path_buf();
-    records_path_dir.pop();
-    fs::create_dir_all(&records_path_dir)
-        .unwrap_or_else(|_| panic!("Failed to create directory {}", records_path_dir.display()));
-    let records_file = File::create(&records_path).unwrap();
-    let mut ser = serde_json::Serializer::new(records_file);
-    let mut seq = ser.serialize_seq(None).unwrap();
+    let mut genesis_config = near_config.genesis.config.clone();
+    genesis_config.genesis_height = genesis_height;
+    genesis_config.validators = validators
+        .iter()
+        .map(|(account_id, (public_key, amount))| AccountInfo {
+            account_id: account_id.clone(),
+            public_key: public_key.clone(),
+            amount: *amount,
+        })
+        .collect();
+    genesis_config.validators.sort();
```

Comment:
Rather than implementing `Ord`, you can also `genesis_config.validators.sort_by_key(|it| (&it.account_id, &it.public_key, &it.amount));`. Though I think `Ord` is fine.

Reply:
I agree, `sort_by_key` is even better.